package.json

```diff
@@ -16,16 +16,16 @@
     "bugs": {
         "url": "https://github.com/eventbrite/eventbrite-sdk-javascript/issues"
     },
-    "homepage":
-        "https://github.com/eventbrite/eventbrite-sdk-javascript#readme",
+    "homepage": "https://github.com/eventbrite/eventbrite-sdk-javascript#readme",
     "license": "MIT",
     "scripts": {
-        "check:static": "npm-run-all --parallel lint 'tsc --noEmit'",
+        "check:static": "npm-run-all --parallel lint tsc",
         "format": "prettier-eslint --write",
         "lint": "eslint --cache --max-warnings 0 --ext .ts,.js src",
         "precommit": "lint-staged",
         "test": "jest --config=jest.json",
         "test:watch": "yarn test --watch",
+        "build": "tsc --p ./tsconfig.build.json && babel src --ignore **/*.spec.ts --out-dir lib/cjs --extensions \".ts,.tsx\"",
```

Comment:
Can you break this up into two `build:*` scripts for readability. And _then_ we can also do `npm-run-all build:A build:B`

Reply:
yeah!